### PR TITLE
Add bumpversion

### LINKED ISSUE #28: Install bumpversion to update easily version number

### PR DESCRIPTION
**Pull request by pierrepo**
bumpversion (actually, its replacement `bump2version`) will automatically handle version number updates.

The file `devtools/RELEASE.md` explain the procedure to perform an update.

A `Makefile` now eases tests and linting.

Fix #28

### CODE REVIEW

**Review by HubLot — APPROVED** [review]
Looks good to me!

File: devtools/RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+```
+$ bump2version --verbose --config-file devtools/bumpversion.cfg minor
+```
+
+For major update (x.y.z → **x+1**.0.0):
+```
+$ bump2version --verbose --config-file devtools/bumpversion.cfg major
+```
+
+Remarque :
```
Comment (patrickfuchs) [inline devtools/RELEASE.md:43]: Remark:

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+lint: ## Lint code
+	pycodestyle buildh \
+	&& pydocstyle buildh \
+	&& pylint buildh
+.PHONY: lint
+
+
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```
Comment (patrickfuchs) [inline Makefile:26]: I don't understand that one. I'm not very familiar with Makefile, what does `$(MAKEFILE_LIST)` contains?

File: devtools/RELEASE.md (hunk 1 above)
Comment (pierrepo) [reply devtools/RELEASE.md]: Fixed ;-)

**Review by patrickfuchs — COMMENTED** [review]
Thanks @pierrepo. I spotted one english typos and have a question on the Makefile. I don't claim it's wrong, it's more for me to understand 😉. Other question about the Makefile, it's there to ease the different tasks we have to do for releasing a new version, right?

File: Makefile (hunk 2 above)
Comment (pierrepo) [reply Makefile:26]: `$(MAKEFILE_LIST)` corresponds to the [name](https://ftp.gnu.org/old-gnu/Manuals/make-3.80/html_node/make_17.html) of the Makefile itself.

`help` is the default rule for the Makefile and produces a summary of available rules:
```
$ make
install-env                    Install conda environement
lint                           Lint code
tests                          Run tests
update-env                     Update conda environement
```

Comment (patrickfuchs) [reply Makefile:26]: Thanks 👍

**Pull request merged**